src/components/card/CardSidebarTabDetails.vue

```diff
@@ -22,6 +22,19 @@
 
 <template>
 	<div v-if="copiedCard">
+		<div class="section-wrapper">
+			<div class="section-details">
+				<div class="button-group">
+					<button class="icon-checkmark" @click="changeCardDoneStatus()">
```

Comment:
These should be `NcButton` components, see https://nextcloud-vue-components.netlify.app/#/Components/NcButton